### PR TITLE
Сomparison dropdown styles

### PR DESCRIPTION
**Pull request by StasDachinsky**
## Brief
- https://github.com/pyroscope-io/pyroscope/issues/1445
## Changes
- added comparison tag selector scroll
- added comparison tag selector text-overflow
- added tests for 'selected' state and dropdown close 
## Concerns
-

https://user-images.githubusercontent.com/23450818/187423387-edfb1704-4550-473d-9a97-80bfb586a5ef.mov

### CODE REVIEW

**Review by pavelpashkovsky — COMMENTED** [review]
LGTM

File: webapp/javascript/pages/tagExplorer/components/TagsSelector.module.scss [hunk 1]
```diff
@@ -23,6 +24,10 @@
       padding: 10px 30px;
       width: 100%;
       text-align: left;
+      white-space: nowrap;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      max-width: 100%;
```
Comment (dogfrogfog) [inline webapp/javascript/pages/tagExplorer/components/TagsSelector.module.scss:30]: you are using `width: 100%`, is `max-width` necessary?

**Comment by dogfrogfog** [comment]
👍

**Comment by Rperry2174** [comment]
Looks great so far, can we try making the width adapt to the content inside so that we don't have to cut off part of the tags?

**Comment by StasDachinsky** [comment]
Hey @Rperry2174, which one you prefer?

https://user-images.githubusercontent.com/23450818/187700385-4ccbfe2b-9fab-4f0d-a4cb-8b5b68149dc8.mov

https://user-images.githubusercontent.com/23450818/187700402-167c8cbc-2c16-46fe-b737-bb78991dc207.mov

**Review by Rperry2174 — APPROVED** [review]

**Pull request merged**